conans/client/cache/remote_registry.py

```diff
@@ -267,6 +268,16 @@ def _validate_url(self, url):
         else:
             self._output.warn("The URL is empty. It must contain scheme and hostname.")
 
+    @contextmanager
+    def _editables_metadata_from_cache(self):
+        """
+        Hide editable packages to get the cache layout instead of the editable one
+        """
+        editables = self._cache.editable_packages
+        self._cache.editable_packages = {}
```

Comment:
The member `editable_packages` is not a dictionary, it is an instance of `EditablePackages` class, we should override the `_edited_refs` member. I would like to keep the same type, one never knows if there is a `isinstance` check somewhere.